public/video-ui/src/components/TagSearch/TagSearch.jsx

```diff
@@ -12,10 +13,9 @@ class TagSearch extends React.Component {
       nextProps.selectedTagIndex !== null &&
       prevProps.selectedTagIndex !== nextProps.selectedTagIndex
     ) {
-      const listNode = this.refs.list;
-      const elementHeight = listNode.children[0].offsetHeight;
-      if (listNode) {
-        listNode.scrollTop =
+      if (this.listNodeRef.current) {
```

Comment:
This was the least trivial fix. I hadn't looked at this component before, but it seems that this logic is in place to make sure that the currently focused tag element is in view when you're navigating through the list using the arrow keys:

<img width="651" height="337" alt="Image" src="https://github.com/user-attachments/assets/494854c3-ad44-47f2-a761-021643baf0d8" />

I've tested the new code locally and the scroll behaviour seems to be the same as it was before 👍